app/controllers/comment_controller.php

```diff
@@ -0,0 +1,28 @@
+<?php 
+Class CommentController extends AppController
+{
+     public function most_liked()
+    {
+        $comments = Comment::get_most_liked();
```

Comment:
Methods inside the model should be camelCase and helpers or functions should be snake_case po :bow: